obs-packaging/create-repo-branch.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+[ -z "${DEBUG}" ] || set -o xtrace
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+script_name="$(basename "${BASH_SOURCE[0]}")"
+
+# shellcheck source=./../scripts/lib.sh
+source "${script_dir}/../scripts/lib.sh"
+
+# shellcheck source=scripts/obs-docker.sh
+source "${script_dir}/scripts/obs-docker.sh"
+
+readonly home_project="home:katacontainers"
+readonly template_pkg="kata-pkg-template"
+arch_target=${ARCH:-$(uname -m)}
+
+# shellcheck source=scripts/obs-docker.sh
+source "${script_dir}/scripts/obs-pkgs.sh"
+
+pkg_exist() {
+	local project="$1"
+	local pkg="$2"
+
+	docker_run osc list "${project}" | grep "${pkg}" || return 1
+	return 0
+
+}
+
+# Array of repositories.
+#
+# Each element is comprised of mutiple parts in the form:
+#
+#   name::project::repository
+#
+typeset -a repos
+repos+=(CentOS_7::CentOS:CentOS-7::standard)
+repos+=(Fedora_26::Fedora:26::standard)
+repos+=(Fedora_27::Fedora:27::standard)
+repos+=(Fedora_28::Fedora:28::standard)
+repos+=(RHEL_7::RedHat:RHEL-7::standard)
+repos+=(SLE_12_SP3::SUSE:SLE-12-SP3:GA::standard)
+repos+=(openSUSE_Leap_42.3::openSUSE:Leap:42.3::standard)
+repos+=(xUbuntu_16.04::Ubuntu:16.04::universe)
+repos+=(xUbuntu_17.10::Ubuntu:17.10::universe)
+repos+=(xUbuntu_18.04::Ubuntu:18.04::universe)
+
+# Array of mantainers.
+#
+# Each element is comprised of mutiple parts in the form:
+#
+#   userid::role
+#
+typeset -a mantainers
+mantainers+=(jcvenega::maintainer)
```

Comment:
- Shouldn't this be:

   ```
   maintainers+=(jcvenega::maintainer)
   ```

- Are these "top-level" maintainers, or could they be architecture-specific ones? I'm wondering if you could add a few more users, specifically to handle ARM + PPC64le?

Reply:
@jcvenegas : Can you please update the PR with the mentioned changes which look fairly simple. I can then test it on Power System and check if any arch specific changes are still needed to generate packages for ppc64le.